pico-statsd/src/main/scala/org/pico/statsd/syntax/event/package.scala

```diff
@@ -49,13 +49,14 @@ package object event {
     @inline
     def withSimpleTimer(aspect: String, sampleRate: Option[Double], tags: String*)
                    (implicit c: StatsDClient): SinkSource[A, B] = {
```

Comment:
The performance optimisation is here.  The `toArray` is pre calculated, so that it doesn't have to be recalculated on every message.